Makefile

```diff
@@ -8,10 +8,10 @@ NDK_BUILD = ndk-build
 WGET      = wget
 
 BASEURL   = https://github.com/measurement-kit/measurement-kit/releases/download
-VERSION   = v0.1.2
+VERSION   = v0.2.0-beta
 TAG       =
 INPUT     = measurement_kit-jni-$(VERSION)$(TAG).tar.bz2
-OVERSION  = $(VERSION).2
+OVERSION  = $(VERSION)-android-1
```

Comment:
@lorenzoPrimi here I've added an additional `android` tag to the version to indicate that what follows is the version of Android code, but I'm not sure this is better than before in terms of characterizing the version. Your thoughts?